wolfssl-sys/build.rs

```diff
@@ -157,8 +157,54 @@ fn build_wolfssl(wolfssl_src: &Path) -> PathBuf {
     }
 
     if build_target::target_arch().unwrap() == build_target::Arch::ARM {
-        // Enable ARM ASM optimisations
-        conf.enable("armasm", None);
+        // Enable ARM ASM optimisations, except for android armeabi-v7a
+        if build_target::target_os().unwrap() != build_target::Os::Android {
+            conf.enable("armasm", None);
+        }
+    }
+
+    if build_target::target_os().unwrap() == build_target::Os::Android {
+        // Build options for Android
+        let (chost, arch_flags, arch, configure_platform) =
```

Comment:
Was `chost` a typo in the `expressvpn/lightway-core` version I wonder...?

Reply:
I think it means "configure host", as the term `host` has a different meaning in configure.

Here is a pretty good summarization:
https://github.com/lu-zero/autotools-rs/blob/27d0437e071ddbf68ff0bbacfa02e01a04cc6b42/src/lib.rs#L69-L106